pkgs/sagemath-doc-html/meson.build

```diff
@@ -0,0 +1,21 @@
+project('sagemath-doc-html',
+        version: run_command('cat',
+                             join_paths(meson.current_source_dir(), 'VERSION.txt'),
+                             check: true).stdout().strip()
+)
+
+docdir = get_option('datadir') / 'share' / 'doc' / 'sage'
+
+html = custom_target('HTML documentation',
+  output: 'index.html',
+  command: [join_paths(meson.current_source_dir(), 'doc', 'make.sh'),
```

Comment:
I think the `join_paths` calls are not needed

Reply:
what do you suggest to use instead?

Reply:
`/`

Reply:
thanks, done